src/modules/shared/pages/SelectModule.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="row q-col-gutter-xl q-pa-lg flex items-center module-selector bg-grey-8">
+    <img src="../../../assets/logos/kriptone-logo.svg" alt="" class="logo" />
+    <div class="col-6">
+      <q-card class="card-selection" @click="vulnerabilityHandler">
+        <q-card-title> Kriptone Tools </q-card-title>
```

Comment:
`[Nit]` Spelling mistake


```suggestion
        <q-card-title> Kriptome Tools </q-card-title>
```